src/dlgConnectionProfiles.cpp

```diff
@@ -508,8 +508,8 @@ void dlgConnectionProfiles::slot_deleteprofile_check( const QString text )
 // actually performs the deletion once the correct text has been entered
 void dlgConnectionProfiles::slot_reallyDeleteProfile()
 {
-    QString profile = profiles_listwidget->currentItem()->text();
-    QDir dir( QStringLiteral( "%1/.config/mudlet/profiles/%2" ).arg( QDir::homePath() ).arg( profile ) );
+    QString profile = profiles_tree_widget->currentItem()->text();
```

Comment:
Um, as it is a `QListWidget` the old name (which I think is the result of a rename __I__ did in the _ugly_ commit 😊 ) is the one to use and I understand this is what is causing the Travis C.I. build fail... 😭